CRAN_questions.md

```diff
@@ -0,0 +1,19 @@
+This document briefly presents some questions this working group has discussed about CRAN to help it being successful.
+
+ * CRAN does an amazing amount of work. What are the most critical issues that CRAN is facing?
+ * CRAN has worked relatively similar for the first 20 years. What is the future of CRAN for the next 20 years?
+ * There is some tension between packages' developers and users' interests respect to checks. What is the purpose of CRAN checks? 
+ * What other are important aspects people should have in mind when volunteering to help CRAN?
+ * Currently there are two repositories with agreements to avoid name conflicts, CRAN and Bioconductor. Would CRAN support the development of additional repositories along the lines of Bioconductor? What kinds of policies would that involve?
+ * The CRAN team is very few people volunteering and lots of pressure. How could people and outside groups like us help?
```

Comment:
Again, I would avoid presuming that they are under a lot of pressure. We could just ask how the community in general could help CRAN.

Reply:
Yes, it is better not presuming anything. I removed the sentence.